_sources/Unit5-Writing-Classes/topic-5-1-parts-of-class.rst

```diff
@@ -280,20 +407,37 @@ To call a method to do its job, we create an object of the class and then use th
 Object-Oriented Design
```

Comment:
Should the OOD subsection be earlier? Not sure why I put it at the end. 
Also, the new draft (https://apcentral.collegeboard.org/media/pdf/ap-computer-science-a-draft-course-framework.pdf) has LOs 3.1 and 3.2 about abstraction and design, if you want to work any of that vocab in.

Reply:
Actually, I think this section is good here. It's a bit esoteric with it's discussion of UML and whatnot. I wouldn't want to make it too prominent.